nspl/ds/Set.php

```diff
@@ -250,7 +251,7 @@ public static function fromArray(array $array)
      */
     public function toArray()
     {
-        return array_values($this->array);
+        return is_null($this->array) ? [] : array_values($this->array);
```

Comment:
This change is redundant since we already initialized ```$this->array``` in the constructor. It is a protected property and it shouldn't became anything else, unless we make a mistake.